CatAnalyzer/test/customise_cff.py

```diff
@@ -0,0 +1,27 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
in this version, I will remove the "customise_cff.py". because we have a option for using DATA.
we will using "LumiMask".